packages/rum/test/benchmarks/server.js

```diff
@@ -39,28 +39,28 @@ function generateImageUrls(port, number) {
   return urls
 }
 
-/**
- * Strip license and sourcemap url
- */
-const APM_BUNDLE = getMinifiedApmBundle()
-  .replace(
-    '/*! For license information please see elastic-apm-rum.umd.min.js.LICENSE */',
-    ''
-  )
-  .replace('//# sourceMappingURL=elastic-apm-rum.umd.min.js.map', '')
-
 /**
  * Adding a random value at the end of the script text prevents
  * Chrome from caching the parsed/JITed script
  */
-function getRandomBundleContent() {
-  let content = APM_BUNDLE
+function getRandomBundleContent(apmBundle) {
+  let content = apmBundle
   content += `var scriptId = ${Date.now()};`
   return content
 }
 
-module.exports = function startServer() {
+const startServer = (module.exports = filename => {
```

Comment:
I think it's cleaner to define the `startServer` function and then assign it to the exports.